sensor/kubernetes/eventpipeline/output/output_impl.go

```diff
@@ -0,0 +1,63 @@
+package output
+
+import (
+	"github.com/stackrox/rox/generated/internalapi/central"
+	"github.com/stackrox/rox/pkg/logging"
+	"github.com/stackrox/rox/sensor/common/detector"
+	"github.com/stackrox/rox/sensor/kubernetes/eventpipeline/component"
+)
+
+var (
+	log = logging.LoggerForModule()
+)
+
+type outputQueueImpl struct {
+	innerQueue   chan *component.ResourceEvent
+	forwardQueue chan *central.MsgFromSensor
+	detector     detector.Detector
+}
+
+// Send sends a ResourceEvent message to the inner queue
```

Comment:
The function name is part of the godoc text
```suggestion
// Send a ResourceEvent message to the inner queue
```